pkg/csi/cinder/controllerserver.go

```diff
@@ -578,6 +578,14 @@ func (cs *controllerServer) ControllerExpandVolume(ctx context.Context, req *csi
 		return nil, status.Errorf(codes.Internal, fmt.Sprintf("Could not resize volume %q to size %v: %v", volumeID, volSizeGB, err))
 	}
 
+	// we need wait for the volume to be available or InUse, it might be error_extending in some scenario
+	targetStatus := []string{openstack.VolumeAvailableStatus, openstack.VolumeInUseStatus}
+	err = cs.Cloud.WaitVolumeTargetStatus(volumeID, targetStatus)
+	if err != nil {
+		klog.V(3).Infof("Failed to WaitVolumeTargetStatus: %v", err)
```

Comment:
Why`Info` is used here? Isn't it an error?

Reply:
there are various place need update ... 
for here, yes , need update